packages/db/postgres/functions.sql

```diff
@@ -120,48 +120,39 @@ CREATE OR REPLACE FUNCTION create_upload(data json) RETURNS TEXT
 AS
 $$
 DECLARE
-  backup_url TEXT;
   inserted_upload_id BIGINT;
+
 BEGIN
   -- Set timeout as imposed by heroku
   SET LOCAL statement_timeout = '30s';
 
   PERFORM create_content(data);
 
-  insert into upload (user_id,
+  insert into upload as upld (user_id,
                       auth_key_id,
                       content_cid,
                       source_cid,
                       type,
                       name,
                       inserted_at,
-                      updated_at)
+                      updated_at,
+                      backup_urls)
   values ((data ->> 'user_id')::BIGINT,
             (data ->> 'auth_key_id')::BIGINT,
             data ->> 'content_cid',
             data ->> 'source_cid',
             (data ->> 'type')::upload_type,
             data ->> 'name',
             (data ->> 'inserted_at')::timestamptz,
-            (data ->> 'updated_at')::timestamptz)
+            (data ->> 'updated_at')::timestamptz,
+            data -> 'backup_urls')
   ON CONFLICT ( user_id, source_cid ) DO UPDATE
     SET "updated_at" = (data ->> 'updated_at')::timestamptz,
         "name" = data ->> 'name',
-        "deleted_at" = null
+        "deleted_at" = null,
+        "backup_urls" = (data -> 'backup_urls')::jsonb || upld.backup_urls
```

Comment:
This will replace the backups on chunked uploads, so I think we need to "merge" them here.

Can we add a test?

Reply:
@vasco-santos `||` act as a merge on JSONB

> The || operator concatenates two JSON objects by generating an object containing the union of their keys, taking the second object's value when there are duplicate keys. 

Good catch, I added a test (please see `can backup chunked uploads` in `packages/db/test/backup.spec.js`).

Reply:
wow, that's pretty cool, didn't know || feature as JSONB! I see by the test it is working  😉